pkg/resources/flow.go

```diff
@@ -77,8 +77,7 @@ func (f Flow) Key() string {
 	return "flow/" + f.generatedName
 }
 
-// Triggers the flow deployment like it was the resource creation
-func (f *Flow) Create() error {
+func (f *Flow) constructGraph(destroy bool) (interfaces.DependencyGraph, error) {
```

Comment:
It is split already into Create and Delete. Initially it was Create only. After I wrote Delete it turned out that both methods look very similar and have only minor differences. So I extracted common code into constructGraph method and call it from both places